build/resources/package/chocolatey/livestreamer-twitch-gui.nuspec.tpl

```diff
@@ -26,14 +26,14 @@ Example: `choco uninstall livestreamer-twitch-gui -y -params '"/Purge"'`
   [Chromium]: https://www.chromium.org/ "Chromium"
     </description>
     <projectUrl><%= homepage %></projectUrl>
-    <packageSourceUrl>https://github.com/bastimeyer/livestreamer-twitch-gui</packageSourceUrl>
-    <tags>livestreamer twitch gui streaming</tags>
+    <packageSourceUrl>https://github.com/streamlink/streamlink-twitch-gui</packageSourceUrl>
+    <tags>streamlink twitch gui livestreamer streaming</tags>
     <copyright>Sebastian Meyer</copyright>
-    <licenseUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/blob/master/LICENSE</licenseUrl>
-    <docsUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/wiki</docsUrl>
-    <bugTrackerUrl>https://github.com/bastimeyer/livestreamer-twitch-gui/issues</bugTrackerUrl>
+    <licenseUrl>https://github.com/streamlink/streamlink-twitch-gui/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://github.com/streamlink/streamlink-twitch-gui/wiki</docsUrl>
+    <bugTrackerUrl>https://github.com/streamlink/streamlink-twitch-gui/issues</bugTrackerUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <iconUrl>https://cdn.rawgit.com/bastimeyer/livestreamer-twitch-gui/master/src/img/icon-256.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/streamlink/streamlink-twitch-gui/master/src/img/icon-256.png</iconUrl>
     <dependencies>
       <dependency id="livestreamer" version="1.12.2"/>
```

Comment:
Livestreamer is still being supported... For now at least... I have kept this dependency, because of the missing streamlink package.

Will the streamlink choco install dir be different than the default one chosen by the installer? If so, the path should be added here:
https://github.com/bastimeyer/livestreamer-twitch-gui/blob/eb12b0cfe5e7f82581082ca68b0eeb31791809a8/src/config/streamprovider.json#L17-L60

> Notable that this will leave the package in a non-functional state (at least on its own), but I don't want to hold up the PR.

What do you mean with non-functional? The old `livestreamer-twitch-gui` package doesn't need to be upgraded (yet). This can be done once `streamlink` and `streamlink-twitch-gui` have been registered and published. I'll let you know when I have moved the repo and released `1.0.0` (during this WE, no need to rush).